### PR TITLE
Add UI skeleton with User page

### PR DESCRIPTION
**Pull request by dash102**
- Sets up each page to use a universal header
- Creates a profile.html page for a hardcoded user

Not yet totally styled (as evidenced by the horrific look), but will be uniformized with other pages in a future PR

image

### CODE REVIEW

**Comment by nikhitashanker** [comment]
LGTM

File: project/src/main/webapp/profile.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+    ></script>
+
+    <script> 
+      $(function(){
```
Comment (nikhitashanker) [inline project/src/main/webapp/profile.html:25]: Maybe move this to the script.js file.

Comment (hlz03) [reply project/src/main/webapp/profile.html:25]: +1

Comment (dash102) [reply project/src/main/webapp/profile.html]: Done.

**Pull request merged**